addon/authenticators/oauth2-password-grant.js

```diff
@@ -276,6 +298,13 @@ export default BaseAuthenticator.extend({
     });
   },
 
+  _tokenRefreshOffset() {
+    const min = this.get('minRefreshOffset');
+    const max = this.get('maxRefreshOffset');
+    Ember.assert('minRefreshOffset cannot be set greater than maxRefreshOffset', min < max);
+    return (Math.floor(Math.random() * min) + (max - min)) * 1000;
+  },
```

Comment:
Sorry for the delay. I have updated the PR with changing it into a computed property.